packages/ui/src/lib/tooltip/tooltip-store.ts

```diff
@@ -0,0 +1,15 @@
+import { writable } from 'svelte/store';
+
+export const tooltipHidden = writable(false);
+
+window.addEventListener('menu-closed', () => {
+  tooltipHidden.update(() => false);
+});
+
+window.addEventListener('menu-open', () => {
+  tooltipHidden.update(() => true);
+});
+
+window.addEventListener('contextmenu', () => {
+  tooltipHidden.update(() => true);
+});
```

Comment:
this store should not listen events thrown by Podman Desktop core

ui package can be used outside of Podman Desktop core

either 
1. it's a generic event ('tooltip-hide', 'tooltip-show' and then Podman Desktop listen on the menu items and send tooltip-hide and tooltip-show)
2.  there is a custom Tooltip only in the core that extends ui Tooltip